portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,9 +55,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("text-input");
+     UserService userService = UserServiceFactory.getUserService();
+     Entity commentEntity = new Entity("Comments");
+     String username;
+    if (userService.isUserLoggedIn()){
+        username = userService.getCurrentUser().getEmail(); 
+        commentEntity.setProperty("username", username);  
```

Comment:
Since the username is not re-used anywhere I think you can simplify this to 

```java
if (userService.isUserLoggedIn()) {
    commentEntity.setProperty("username", userService.getCurrentUser().getEmail());
}
```